model/jpa/src/main/resources/META-INF/jpa-changelog-8.0.0.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!--
+  ~ * Copyright 2018 Red Hat, Inc. and/or its affiliates
+  ~ * and other contributors as indicated by the @author tags.
+  ~ *
+  ~ * Licensed under the Apache License, Version 2.0 (the "License");
+  ~ * you may not use this file except in compliance with the License.
+  ~ * You may obtain a copy of the License at
+  ~ *
+  ~ * http://www.apache.org/licenses/LICENSE-2.0
+  ~ *
+  ~ * Unless required by applicable law or agreed to in writing, software
+  ~ * distributed under the License is distributed on an "AS IS" BASIS,
+  ~ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ * See the License for the specific language governing permissions and
+  ~ * limitations under the License.
+  -->
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet author="keycloak" id="8.0.0">
+        <customChange class="org.keycloak.migration.jpa.LazyCreateIndexCustomChange">
+            <param name="tableName" value="EVENT_ENTITY" />
+            <param name="indexName" value="IDX_EVENT_TIME" />
+            <param name="columns" value="EVENT_TIME" />
```

Comment:
> I think for most queries, indexing on event_time is enough. The resulting set after it is filtered by event_time is small enough that it would work.

Please add the REALM_ID column to the index (and put it first).
It may not matter much if you only have a handful of realms, but I'm working on an environment with 600+ realms (which will increase further over the next few years to about a thousand).